romancal/outlier_detection/outlier_detection.py

```diff
@@ -230,6 +235,8 @@ def blot_median(self, median_model):
 
         log.info("Blotting median")
         for model in self.input_models:
+            # TODO: fix dtype conflict
+            model.dq = model.dq.astype("uint32")
```

Comment:
Do you intend to have a # TODO here?

Reply:
It seems that the romanisim files that I used do not suffer from the `int32`/`uint32` issue, so I removed the `TODO` label. Thanks for the catch!